webapi/src/main/scala/org/knora/webapi/messages/StringFormatter.scala

```diff
@@ -1969,6 +1986,14 @@ class StringFormatter private (
       case None    => None
     }
 
+  def validateAndEscapeOptionalProjectIri(
+    maybeString: Option[String]
```

Comment:
As above with the `validateAndEscapeOptionalIri` I do think it is worth to consider removing these methods which operate on an `Option[String]` for the same reasoning.

Reply:
All "optional" methods are gone now 🔥